src/components/quality-menu/quality-menu.js

```diff
@@ -132,18 +134,19 @@ class QualityMenu extends Component {
 
     // Progressive playback doesn't support auto
     if (qualityOptions.length > 1 && props.player.streamType !== 'progressive') {
-      const activeTrack: Object = qualityOptions.find(track => track.value.active === true).value;
+      const activeTrack: Object = qualityOptions.find(track => track.value.active === true)?.value;
+      const label = activeTrack?.label || DEFAULT_VIDEO_TRACK_LABEL;
       let qualityLabel;
       if (rtlLanguages.includes(this.props.player._localPlayer._config.ui.locale)) {
-        qualityLabel = activeTrack.label + ' - ' + this.props.qualityAutoLabelText;
+        qualityLabel = label + ' - ' + this.props.qualityAutoLabelText;
       } else {
-        qualityLabel = this.props.qualityAutoLabelText + ' - ' + activeTrack.label;
+        qualityLabel = this.props.qualityAutoLabelText + ' - ' + label;
       }
       qualityOptions.unshift({
         label: this.props.qualityAutoLabelText,
         dropdownOptions: {
           label: qualityLabel,
-          badgeType: getLabelBadgeType(activeTrack.height)
+          badgeType: getLabelBadgeType(activeTrack?.height || 360)
```

Comment:
I feel like `360` should be in const instead of '360p' and label be like:
```const label = activeTrack?.label || `${DEFAULT_VIDEO_TRACK}px`;```
If default label changed from 360 to any other value label will takes updated value automatically.

Reply:
got your point- fixed.